web/src/components/View/View.css

```diff
@@ -19,8 +19,17 @@
 
 .view-container__title {
     color: var(--colarkfontcolor);
+<<<<<<< HEAD
```

Comment:
do we want to keep the <<< HEAD ===== >>>>> lines

Reply:
nope!